smart-contract-verifier/smart-contract-verifier-http/config/base.toml

```diff
@@ -23,6 +23,9 @@ list_url = "https://solc-bin.ethereum.org/linux-amd64/list.json"
 ## The only required field for the s3 fetcher
 #bucket = "bucket"
 
+# [solidity.extensions.sig_provider]
+# url = "http://127.0.0.1:9000/"
```

Comment:
Nit: I suggest to make it the other way around. Not to specify extensions as part of solidity, vyper, and sourcify settings, but specify solidity, vyper, and sourcify as different kinds of extensions. E.g.
```
[extensions.vyper.sig_provider]
url = "0.0.0.0:1234"

[extensions.sourcify.sig_provider]
url = "0.0.0.0:1234"
```

Grouping them together makes them independent from the actual verifier settings. It makes it easier to remove them, and seems to be more explicit for me. Furthermore, that way we extend the current settings struct without modification of any of its field struct that seems more clear way to do that.